src/components/AnalysisPage/3dModelViewer/setupThreeScene.js

```diff
@@ -81,6 +82,15 @@ const setupThreeScene = (canvas) => {
 
       const mesh = new THREE.Mesh(geometry, material);
 
+      // this if statement has to be seperate from similar statement
+      // on line 76 because the materials have to be defined before the mesh
```

Comment:
Try to avoid specifying particular line numbers in code because if you add a new feature in your next PR that pushes the code down, the correct code won't line up with this comment anymore. Try to just use descriptions instead